components/gcp-click-to-deploy/src/Gapi.ts

```diff
@@ -50,21 +55,18 @@ export default class Gapi {
   public static iam = class {
 
     public static async createKey(projectId: string, serviceAccounts: string) {
-        await this._load();
-        return gapi.client.projects.serviceAccounts.keys.create({
-            name: `projects/${projectId}/serviceAccounts/${serviceAccounts}`
-        }).then(response => response.result,
-                badResult => {
-          throw new Error('Errors creating key for serviceAccount ${serviceAccounts}');
+      await Gapi.load();
+      return gapi.client.request({
+          method: 'POST',
+          path: `https://iam.googleapis.com/v1/projects/${projectId}/serviceAccounts/${serviceAccounts}/keys`,
+        }).then(response => {
```

Comment:
nit: you can simplify this to one line: `.then(response => response.result as CreateKeyResponse)`